### PR TITLE
Completed javascript-primitives-and-variables.

### PR DESCRIPTION
**Pull request by youssefnajjarine**
image

### CODE REVIEW

File: javascript-primitives-and-variables/main.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+var fullName = 'Youssef Najjarine';
+var isCool = false;
+var totalPets = 1;
+var unknown = null;
+var never = ' ';
```
Comment (uzair-ashraf) [inline javascript-primitives-and-variables/main.js:5]: not a string with an empty space, you need to just decalre it.

File: javascript-primitives-and-variables/main.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+console.log(fullName);
+console.log(isCool);
+console.log(totalPets);
+console.log(unknown);
+console.log(never);
+console.log(typeof 'Youssef Najjarine');
+console.log(typeof false);
+console.log(typeof 1);
```
Comment (uzair-ashraf) [inline javascript-primitives-and-variables/main.js:13]: Use typeof with the variables.

**Review by uzair-ashraf — CHANGES_REQUESTED** [review]

File: javascript-primitives-and-variables/main.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+var fullName = 'Youssef Najjarine';
+var isCool = false;
+var totalPets = 1;
+var unknown = null;
+var never =
```
Comment (uzair-ashraf) [inline javascript-primitives-and-variables/main.js:5]: This is a syntax error, please run your code before submitting.

**Review by uzair-ashraf — CHANGES_REQUESTED** [review]

File: javascript-primitives-and-variables/main.js (hunk 3 above)
Comment (youssefnajjarine) [reply javascript-primitives-and-variables/main.js:5]: image

It ran fine for me, and in the instructions it said to not assign it anything
image

**Review by uzair-ashraf — APPROVED** [review]

**Pull request merged**